resources/core/values.yaml

```diff
@@ -18,6 +18,7 @@ global:
     dir:
     version: 3b88aaaf
   docs:
+    enabled: false
```

Comment:
right now it's `global.docs.enabled`, you have to either change this file or requirements.yaml in order to install kyma without those clusterassetgroups